src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -333,13 +333,13 @@ public static bool IsStaticallyDeclared(SyntaxToken token)
                 parentNode = parentNode!.Parent!.Parent;
 
                 // Check if this is a field constant declaration 
-                if (parentNode.GetModifiers().Any(SyntaxKind.ConstKeyword))
+                if (parentNode!.GetModifiers().Any(SyntaxKind.ConstKeyword))
```

Comment:
GetModifiers moved around nullable contexts.  wasn't sure the best thing do to here.  However, given the heavy use of `!` already, this seemed like an acceptable quickpatch.

Reply:
specifically, i didn't want to change runtime behavior.  so i just suppressed to keep things safe.